flake.nix

```
@@ -179,20 +180,25 @@
                 (builtins.readFile "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt")
             )
           ];
+          # Required for the UTF-8 locale to exist!
+          extraCommands = ''
+            mkdir -p usr/lib/locale
+            cp ${pkgs.glibcLocales}/lib/locale/locale-archive usr/lib/locale/locale-archive
```

```suggestion
            cp -a ${pkgs.glibcLocales}/lib/locale/locale-archive usr/lib/locale/locale-archive
```
Not sure if permissions should be maintained, but just in case?